crates/ruma-events/src/room/message/audio.rs

```diff
@@ -88,6 +91,21 @@ impl AudioMessageEventContent {
     pub fn info(self, info: impl Into<Option<Box<AudioInfo>>>) -> Self {
         Self { info: info.into(), ..self }
     }
+
+    /// Returns the caption for the audio as defined by MSC2530.
+    ///
+    /// In short, this is the `body` field if the `filename` field exists and has a different value,
+    /// otherwise the media files does not have a caption.
```

Comment:
Since there is a single file per event, this makes more sense imo.

```suggestion
    /// otherwise the media file does not have a caption.
```